### PR TITLE
rename threshold to threshold_percent

### PR DESCRIPTION
**Pull request by agramfort**
thershold was super misleading. I thought it means a threshold value of connectivity entries but it's actually a percentage of edges to keep. Also I made the default 0.2 as 1 makes little sense (all degrees would be n_nodes).

cc @dengemann @larsoner

### CODE REVIEW

File: examples/connectivity/plot_mne_inverse_envelope_correlation.py [hunk 1]
```diff
@@ -86,7 +86,8 @@
 # Compute the degree and plot it
 # ------------------------------
 
-degree = mne.connectivity.degree(corr, 0.15)
+threshold_percent = 0.15  # percentage of strongest edges to keep in the graph
```
Comment (larsoner) [inline examples/connectivity/plot_mne_inverse_envelope_correlation.py]: This is not a percentage but a proportion

File: mne/connectivity/utils.py [hunk 2]
```diff
@@ -44,16 +44,17 @@ def seed_target_indices(seeds, targets):
...
     """Compute the undirected degree of a connectivity matrix.
 
     Parameters
     ----------
     connectivity : ndarray, shape (n_nodes, n_nodes)
         The connectivity matrix.
-    threshold : float
-        The proportion of activations to keep before computing
-        the degree.
+    threshold_percent : float
```
Comment (larsoner) [inline mne/connectivity/utils.py]: Mixture of proportion / percent is not good (one is 100x the other)

File: examples/connectivity/plot_mne_inverse_envelope_correlation.py (hunk 1 above)
Comment (agramfort) [reply examples/connectivity/plot_mne_inverse_envelope_correlation.py]: what do you suggest? put 15 and not 0.15?

Comment (mmagnuski) [reply examples/connectivity/plot_mne_inverse_envelope_correlation.py]: I prefer `0.15` so maybe `threshold_proportion` or `threshold_prop`? (although I also frequently refer to proportional values as percentages, so although it is not strictly correct it seems intuitive to me)

**Comment by agramfort** [comment]
this would work for me

>

**Comment by larsoner** [comment]
I'm fine with `threshold_prop`

**Pull request merged**